sdk/src/fee_calculator.rs

```diff
@@ -9,6 +9,45 @@ pub struct FeeCalculator {
     // cluster processing load.
     pub lamports_per_signature: u64,
 
+    // What portion of collected fees are to be destroyed, as a fraction of std::u8::MAX
+    pub burn_percent: u8,
```

Comment:
Does `burn_percent` even belong in `FeeCalculator` anymore?    Seems like it's just a `FeeRateGovernor` thing

Reply:
I need to check upstream callers. It was originally in `FeeRateGoverner` but I hit a test that checked it under BPF.  It doesn't really seem like something app developers _should_ need.

One plus of leaving it in `FeeCalculator` would be that, if it becomes a dynamic value in the future, the slot-dependent values are together.

Reply:
When burn_percent changes, I can't think of why we'd ever want it to be slot-specific.  At worst it would change on an epoch boundary, tweaked by a magic via https://github.com/solana-labs/solana/blob/021d0a46f8b107cd7dc79144cd6deeacd376fe85/runtime/src/bank.rs#L460